src/store/module/catalogue.js

```diff
@@ -0,0 +1,19 @@
+export default {
+    state: {
+        catalogue: [],
+    },
+
+    getters: {
```

Comment:
getters are computed, use them in case you need some computation with the data,
like some filtered result or so
instead try using ...mapState